modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -34,6 +35,119 @@ admin_manual:release_notes.adoc
+See the xref:admin_manual:installation/system_requirements.adoc[system requirements] for more information.
+
+NOTE: If you're using the official Docker containers or the Univention appliance, this has been taken care of already.
+
+=== Support for MariaDB 10.6
+
+Version 10.9 adds support for MariaDB 10.6 (https://github.com/owncloud/core/pull/39286[#39286]). If you're planning a database update to a MariaDB version > 10.5, it is **very important to prepare the ownCloud installation first**. Please read the https://doc.owncloud.com/server/latest/admin_manual/maintenance/upgrading/database_upgrade.html[documentation] on this matter carefully before upgrading to MariaDB 10.6.
+
+=== Highly Improved Initial Sync and Discovery Performance
+
+Server 10.9 comes with the means to drastically reduce the time needed by the ownCloud Clients for the initial discovery of the contents in user accounts, especially for those with many files and huge directory structures. Practically, this means when a user account is initially set up, e.g., in the ownCloud Desktop Client or when larger directory structures are added to an existing account (e.g., via sharing or external storage mounts like Windows Network Drives), the client does not need to check every particular folder anymore, as it used to be previously. Instead, the server will compose a full content listing and provide it to the client ("Streaming PROPFIND with depth=infinity", https://github.com/owncloud/core/pull/38583[#38583]). Additionally, to prevent memory issues on the server side, this content listing will be streamed to the client while it is being created.
+
+This new server-side capability needs changes in the ownCloud Clients to make use of it. At the time of the ownCloud Server 10.9 release, client implementations for iOS, Android and Desktop are in the making but not yet available. The implementations will follow soon with the next client releases.
+
+TIP: If you use the currently existing versions of the ownCloud Clients, you will not yet experience performance gains.
+
+=== Manual File Locking: Ability to Define User Groups that can Unlock Files
+
+Since version 10.5, xref:file-locking-in-the-web-interface[ownCloud supports Manual File Locking] to prevent concurrent changes of files by different users. The feature works in a way that it only allows the users who locked a file to unlock it again (_exclusive file locking_) and it provides a configurable timeout for file locks.
+
+Version 10.9 introduces a new sub-feature that allows administrators to define groups of users that are entitled to unlock locked files, independent of whether they have set the lock initially or not (https://github.com/owncloud/core/pull/38222[#38222]). Users in such groups can regain control over files, e.g., if a lock owner is unavailable and a long timeout for file locks is used. Administrators can find the configuration in the _Manual File Locking_ section of the _General_ admin settings.
```

Comment:
FYI, the feature is documented in (admin and user manual):
https://doc.owncloud.com/server/10.9/admin_manual/configuration/files/manual_file_locking.html
https://doc.owncloud.com/server/10.9/user_manual/files/manual_file_locking.html